CMakeLists.txt

```diff
@@ -8,6 +8,9 @@ include("configs/azure_iot_sdksFunctions.cmake")
 
 getIoTSDKVersion()
 message(STATUS "IoT Client SDK Version = ${IOT_SDK_VERSION}")
+message("C FLAGS: ${CMAKE_C_FLAGS}")
```

Comment:
>message [](start = 0, length = 7)

We can avoid these additional messages for now because it maybe too verbose for simple builds.